apollo-router/src/plugins/authentication/mod.rs

```diff
@@ -128,6 +128,9 @@ struct JWTConf {
 struct JwksConf {
     /// Retrieve the JWK Set
     url: String,
+    /// Polling interval in seconds for each JWKS endpoint; defaults to 60
+    #[serde(default="default_poll_interval")]
+    poll_interval: u32,
```

Comment:
Can we use the humantime duration like we do for other duration we have to fill in the configuration ?

Reply:
Sure! Let me look at that and port it.